02/app.js

```diff
@@ -0,0 +1,32 @@
+const divFinder = function(){
+const div = document.querySelector('div');
```

Comment:
trzymaj się odpowiednich odstępów, jeśli mamy zagnieżdżenie to powinien być odstęp od lewe krawędzi